integration_tests/integration/barcode/createNewContactByPrisonNumber.spec.ts

```diff
@@ -4,6 +4,10 @@ import ReviewRecipientsPage from '../../pages/barcode/reviewRecipients'
 import CreateNewContactByPrisonNumberPage from '../../pages/barcode/createNewContactByPrisonNumber'
 
 context('Create New Contact By Prison Number Page', () => {
+  beforeEach(() => {
```

Comment:
Defo one for another day, but one of the reasons we moved all the `cy....` type gubbins out of the test was to abstract the test away from the browser driver, and let the `Page` subclass interact with the browser driver.
That knowledge is now sneaking back into the tests; so I wonder if (some other time) we look at other approaches for this type of thing? Maybe even something simple like creating a superclass for all cypress test specs called `CypressTest` or similar that has this `beforeEach` hook? If nothing else that would remove the duplication?
Just a thought 🤷‍♂️

Reply:
Ah ok. It was a bit of a knee jerk reaction on my behalf because I created a stub which was then cleared by the goTo - took me ages to find the issue so jumped into the quickest fix. Yes let's have another look when we have a bit more time.